lib/view/home_page.dart

```diff
@@ -1,68 +1,129 @@
+    Widget child,
+  ) {
+    return Stack(
+      children: <Widget>[_renderBackgroundContainer(context), child],
+    );
+  }
+
+  Widget _renderList(BuildContext context) {
+    return SliverList(
+      delegate: SliverChildListDelegate([
+        Container(height: 12),
+        CardContainer(
+          title: 'Look After Yourself',
+          cards: staffWelfare,
+        ),
+        CardContainer(
+          title: 'Intubations',
+          cards: intubation,
+        ),
+        CardContainer(
+          title: 'ICU',
+          cards: icu,
+        ),
+        // Make sure the bottom CardContainer has room to breathe.
+        SizedBox(height: 12),
+      ]),
+    );
+  }
+
+  Widget _renderBody(BuildContext context) {
+    var appBarHeight = 70, logoHeight = 24;
+
+    final Widget mainLogo = SvgPicture.asset('assets/images/main_logo.svg',
+        height: logoHeight.toDouble());
+
+    var appBarBottom = appBarHeight.toDouble() - logoHeight.toDouble();
+
+    var scrollPos = _scrollPosition.round();
+
+    var percentage = scrollPos < appBarBottom
+        ? ((scrollPos) / appBarBottom.toDouble())
+        : 1.00;
+
+    return Builder(builder: (context) {
+      final _scrollController = PrimaryScrollController.of(context);
+
+      _scrollController.addListener(() {
+        setState(() {
+          _scrollPosition = _scrollController.position.pixels;
+        });
+      });
+
+      return CustomScrollView(
         slivers: <Widget>[
           SliverAppBar(
             automaticallyImplyLeading: false,
-            expandedHeight: 100,
+            expandedHeight: appBarHeight.toDouble(),
             bottom: PreferredSize(
-              preferredSize: Size.fromHeight(50),
+              preferredSize: Size.fromHeight(appBarHeight.toDouble()),
               child: Padding(
                 padding: const EdgeInsets.fromLTRB(12, 0, 12, 0),
                 child: Row(
                   mainAxisAlignment: MainAxisAlignment.spaceBetween,
                   children: <Widget>[
-                    Text(
-                      'Western Health',
-                      style: AppStyles.appBarTextStyle,
-                    ),
+                    mainLogo,
                     IconButton(
-                      icon: Icon(
-                        Icons.info_outline,
-                        color: AppColors.appBarIcon,
-                      ),
+                      icon: Icon(Icons.info_outline,
+                          color: ColorUtils.generateIconColor(
+                              AppColors.homeAppBarIcon,
+                              AppColors.appBarIcon,
+                              percentage)),
                       onPressed: () => InfoView.navigateTo(context),
                     )
                   ],
                 ),
               ),
             ),
-            backgroundColor: AppColors.appBarBackground,
+            backgroundColor: Color.fromRGBO(255, 255, 255, percentage),
```

Comment:
Move this magic color to the AppStyle class as a method?

Reply:
yep!